cli/src/test/java/com/devonfw/tools/ide/context/BashFindingTest.java

```diff
@@ -0,0 +1,38 @@
+package com.devonfw.tools.ide.context;
+
+import org.junit.jupiter.api.Test;
+
+import com.devonfw.tools.ide.os.WindowsHelperMock;
+
+/**
+ * Test for bash finding functionality using WindowsHelper.
+ */
+public class BashFindingTest extends AbstractIdeContextTest {
+
+  @Test
+  public void testFindBashWithWindowsHelper() {
```

Comment:
This test is only testing that the bash can be found. If we are on Windows the implementation will use `WindowsHelper` but otherwise it won't. The test should not care about such implementation secrets.
```suggestion
  public void testFindBash() {
```